aiida/restapi/resources.py

```diff
@@ -212,17 +215,6 @@ class Node(BaseResource):
     _translator_class = NodeTranslator
     _parse_pk_uuid = 'uuid'  # Parse a uuid pattern in the URL path (not a pk)
 
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
-        from aiida.orm import Node as tNode
-        self.tclass = tNode
```

Comment:
I cannot see why it is ok to remove these lines? Where is this being set for the `Node` base resource?

Reply:
I have tried to explain this in my commit message of 41ad616.

It seems these lines were "missed" to be removed when moving to the REST API v4 (AiiDA v1). It was also present for all the sub-classes, but was removed in the transition, and so I guess this acted as a "meta" class in a way for these sub-resources, but is now not relevant, since this information is retrieved in a different way.
However, I am not _completely_ sure and would like a confirmation from @waychal, @asle85 (or @ltalirz who originally wrote these lines some years ago).

Reply:
> (...) Where is this being set for the `Node` base resource?

It is not - but it is also (apparently) never used.

Reply:
I expect by now, since there haven't been any outcry from either the tests (of this) or @waychal @asle85 or @ltalirz that it's fine to remove.

Reply:
I'm not an expert here - I trust Casper's assessment :-)